src/Units/AWBWUnits.java

```diff
@@ -0,0 +1,778 @@
+    private static final int MAX_AMMO = 6;
+    private static final int VISION_RANGE = 4;
+    private static final int MOVE_POWER = 6;
+
+    private static final MoveType moveType = new Flight();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new AWBWWeapons.StealthShots() };
+
+    public StealthModel()
+    {
+      super("Stealth", AWBWUnitEnum.STEALTH, UnitRoleEnum.AIR_ASSAULT, ChassisEnum.AIR_HIGH, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class StealthHideModel extends StealthModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int IDLE_FUEL_BURN = 8;
+
+    public StealthHideModel()
+    {
+      super();
+      type = AWBWUnitEnum.STEALTH_HIDE;
+      idleFuelBurn = IDLE_FUEL_BURN;
+      hidden = true;
+    }
+  }
+
+  public static class BBombModel extends AWBWUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 25000;
+    private static final double STAR_VALUE = 0.6;
+    private static final int MAX_FUEL = 45;
+    private static final int IDLE_FUEL_BURN = 5;
+    private static final int MAX_AMMO = -1;
+    private static final int VISION_RANGE = 1;
+    private static final int MOVE_POWER = 9;
+
+    private static final MoveType moveType = new Flight();
+    private static final UnitActionType[] actions = UnitActionType.BASIC_ACTIONS;
+    private static final WeaponModel[] weapons = {};
+
+    public BBombModel()
+    {
+      super("BBomb", AWBWUnitEnum.BBOMB, UnitRoleEnum.AIR_ASSAULT, ChassisEnum.AIR_HIGH, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+      possibleActions.add(0, new UnitActionType.Explode(5, 3));
+    }
+  }
+
+  // sea
+
+  public static class BBoatModel extends AWBWUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 7500;
+    private static final double STAR_VALUE = 1.0;
+    private static final int MAX_FUEL = 60;
+    private static final int IDLE_FUEL_BURN = 1;
+    private static final int MAX_AMMO = -1;
+    private static final int VISION_RANGE = 1;
+    private static final int MOVE_POWER = 7;
+
+    private static final MoveType moveType = new FloatLight();
+    private static final UnitActionType[] actions = UnitActionType.TRANSPORT_ACTIONS;
+
+    public BBoatModel()
+    {
+      super("BBoat", AWBWUnitEnum.BBOAT, UnitRoleEnum.TRANSPORT, ChassisEnum.SHIP, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER, moveType,
+          actions, new WeaponModel[0], STAR_VALUE);
+      holdingCapacity = 2;
+      holdables = new Vector<ChassisEnum>(Arrays.asList(ChassisEnum.TROOP));
+      possibleActions.add(0, UnitActionType.REPAIR_UNIT);
```

Comment:
Magic 0. Minor instance, but still. I still will normally try to use a named temp.